instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/http/HttpSpanNameExtractor.java

```diff
@@ -70,10 +70,11 @@ static final class Client<REQUEST> implements SpanNameExtractor<REQUEST> {
     @Override
     public String extract(REQUEST request) {
       String method = getter.getHttpRequestMethod(request);
+      String template = getter.getUrlTemplate(request);
       if (method == null || !knownMethods.contains(method)) {
```

Comment:
I think that the template should either be used here or moved down closer to where it's used. We shouldn't bother look up the template if we're just going to return early with `HTTP`.